include/imex/Transforms/Transforms.h

```diff
@@ -0,0 +1,52 @@
+//===- Transforms.h - Transforms Pass declaration ---------------------*- C++
+//-*-===//
+//
+// Copyright 2022 Intel Corporation
+// Part of the IMEX Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file declares all the Transforms Passes.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <llvm/ADT/SmallBitVector.h>
+#include <memory>
+#include <mlir/Analysis/BufferViewFlowAnalysis.h>
+#include <mlir/Conversion/ArithmeticToSPIRV/ArithmeticToSPIRV.h>
+#include <mlir/Conversion/ControlFlowToSPIRV/ControlFlowToSPIRV.h>
+#include <mlir/Conversion/FuncToSPIRV/FuncToSPIRV.h>
+#include <mlir/Conversion/GPUToSPIRV/GPUToSPIRV.h>
+#include <mlir/Conversion/MathToSPIRV/MathToSPIRV.h>
+#include <mlir/Conversion/SCFToSPIRV/SCFToSPIRV.h>
+#include <mlir/Dialect/Affine/IR/AffineOps.h>
+#include <mlir/Dialect/ControlFlow/IR/ControlFlowOps.h>
+#include <mlir/Dialect/Func/IR/FuncOps.h>
+#include <mlir/Dialect/GPU/Transforms/ParallelLoopMapper.h>
+#include <mlir/Dialect/GPU/Transforms/Passes.h>
+#include <mlir/Dialect/MemRef/IR/MemRef.h>
+#include <mlir/Dialect/SCF/IR/SCF.h>
+#include <mlir/Dialect/SPIRV/IR/SPIRVDialect.h>
+#include <mlir/Dialect/SPIRV/IR/SPIRVOps.h>
+#include <mlir/Dialect/SPIRV/IR/TargetAndABI.h>
+#include <mlir/Dialect/SPIRV/Transforms/SPIRVConversion.h>
+#include <mlir/Pass/Pass.h>
+#include <mlir/Target/SPIRV/Serialization.h>
+#include <mlir/Transforms/DialectConversion.h>
+#include <mlir/Transforms/GreedyPatternRewriteDriver.h>
```

Comment:
it is better to move all these includes in cpp file to limit the inclusion scope..